hr_department_sequence/README.rst

```diff
@@ -0,0 +1,38 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+HR Department Sequence
+======================
+
+This module was written to allow you to order department by Parent-Child Relationship and by Sequence Number.
+It also adds some fields on the department like code and active.
```

Comment:
I don't really like the "adds some fields ... like" I prefer to know what I'm installing.

I understand that all fields but those two are used for the parent-child relation. So you could simply remove that "like" and extend a bit with a "It also extends department by adding the two extra fields code and active to ease management with many departments"